investigate.go

```diff
@@ -59,6 +61,16 @@ func InvestigateProjectLicenses(path string) (map[string]float32, error) {
 	for _, file := range files {
 		if !file.IsDir() {
 			fileNames = append(fileNames, file.Name())
+		} else if pureLicenseFileRe.MatchString(strings.ToLower(file.Name())) {
+			// "license" directory, let's look inside
+			subfiles, err := ioutil.ReadDir(paths.Join(path, file.Name()))
+			if err != nil {
```

Comment:
this seems wrong

Reply:
Yep

Reply:
The weird thing is that I tested it on Kotlin and was sure it worked.

Reply:
clearly doesn't though 😄 next time you should make sure to have more unit tests

Reply:
My typical workflow is first develop all the functions, then stabilize at 80-90% coverage and then refactor. This refactoring was not expected so soon (still so awesome to have it!).

Reply:
This workflow makes a lot of sense in the ML/DS world where you have to throw away 30%-50% of the code because you've realized there is a better algorithm - as we have here with google/licenseclassifier.

Reply:
do you think most of this code will be thrown away?